pom.xml

```diff
@@ -110,13 +110,19 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
-      <version>${rest-assured.version}</version>
+      <version>4.3.1</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.sun.xml.bind</groupId>
+          <artifactId>jaxb-osgi</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
What is the purpose of this exclusion? Maybe a comment could be added to explain to future readers why this needs to be excluded.

Reply:
Good point, this was excluded because of version conflict with transitive dependency carried by the Spring's artifact, as I remember. I'll check it again to make sure this is needed for now.

Reply:
I have checked and it looks like currently there is no dependencies conflicts, therefore will remove this. Thank you Marc for noticing this.